.vib/jenkins/goss/jenkins.yaml

```diff
@@ -0,0 +1,16 @@
+command:
+  check-app-jar:
+    exec: java -jar /opt/bitnami/jenkins/jenkins-plugin-manager.jar
+    exit-status: 0
+  check-app-version:
+    exec: java -jar /opt/bitnami/jenkins/jenkins.war --version
+    exit-status: 0
+    stdout:              
+      - {{ .Env.APP_VERSION }}
+file:
+  /opt/bitnami/jenkins/plugins.txt:
+    exists: true
+    contains:
+      - "swarm"
+      - "matrix-auth"
+      - "authorize-project"
```

Comment:
This is where the plugins to download are defined, but I would rather check they are actually present in the plugins folder

Reply:
Alternatively, you can also use the `jenkins-plugin-manager.jar` jar file to assert this